protocol/x/clob/memclob/memclob.go

```diff
@@ -1696,7 +1696,7 @@ func (m *MemClobPriceTimePriority) mustPerformTakerOrderMatching(
 			FillAmount: matchedAmount,
 		}
 
-		success, takerUpdateResult, makerUpdateResult, err := m.clobKeeper.ProcessSingleMatch(ctx, &matchWithOrders)
+		success, takerUpdateResult, makerUpdateResult, _, err := m.clobKeeper.ProcessSingleMatch(ctx, &matchWithOrders)
```

Comment:
order fill event was not being emitted by memclob - skipping

Reply:
FYI MemClob does emit offchain update events (for indexer and grpc streaming) [here](https://github.com/dydxprotocol/v4-chain/blob/0aa8383b7e0176bfaab83f2ac166d0fd408c102d/protocol/x/clob/memclob/memclob.go#L524). It just that this rev share info is not needed in that event